content/400-pulse/200-getting-started.mdx

```diff
@@ -135,13 +135,13 @@ In a project using [Prisma Client](/orm/prisma-client), run the following comman
 npm install @prisma/extension-pulse
 ```
 
-### Store your Cloud Projects API key in your .env file
+### Store your Pulse API key in your .env file
 
-The Pulse extension requires you to use a Cloud Projects API key.
+The Pulse extension requires you to use an API key.
 
 <Admonition type="info">
 
-You should have received an API key from the Cloud Projects dashboard when setting up your project.
+You should have received an API key from the Console Platform dashboard when setting up your project.
```

Comment:
would this be more accurate maybe?

```suggestion
You should have received an API key when you added Prisma Pulse to your project in the Platform Console.
```

also, I think we should add a note in step 2. that an API key will be created at the end.

Reply:
@nikolasburk, I added a note for that in the latest commit.